tests/deprecated_api/test_remove_1-5.py

```diff
@@ -93,6 +93,81 @@ def on_save_checkpoint(self, *args):
         trainer.save_checkpoint(filepath)
 
 
+class BaseSignatureOnLoadCheckpoint(Callback):
+
+    def __init__(self):
+        self.on_load_checkpoint_called = False
+
+
+class NewSignatureOnLoadCheckpoint(BaseSignatureOnLoadCheckpoint):
+
+    def on_save_checkpoint(self, trainer, pl_module, checkpoint) -> dict:
+        return {"something": "something"}
+
+    def on_load_checkpoint(self, trainer, pl_module, checkpoint):
+        assert checkpoint == {"something": "something"}
+        self.on_load_checkpoint_called = True
+
+
+def test_v1_5_0_old_callback_on_load_checkpoint(tmpdir):
+
+    class OldSignature(BaseSignatureOnLoadCheckpoint):
+
+        def on_load_checkpoint(self, callback_state) -> None:
+            assert callback_state is None
+            self.on_load_checkpoint_called = True
+
+    model = BoringModel()
+    trainer_kwargs = {
+        "default_root_dir": tmpdir,
+        "max_epochs": 3,
+    }
+    chk = ModelCheckpoint(save_last=True)
+    trainer = Trainer(**trainer_kwargs, callbacks=[OldSignature(), chk])
+    trainer.fit(model)
+
+    with pytest.deprecated_call(match="old signature will be removed in v1.5"):
+        trainer_kwargs["max_epochs"] = 5
```

Comment:
```suggestion
        trainer_kwargs["max_steps"] = 1
```
one step is sufficient to trigger the warning.

Reply:
Yes, should at lest 4 :)